frontend-react/src/hooks/network/DataDashboard/DataDashboardHooks.ts

```diff
@@ -97,4 +101,45 @@ const useOrgDeliveries = (service?: string) => {
     return { fetchResults, filterManager };
 };
 
-export { useOrgDeliveries };
+const useReportsDetail = (id: string) => {
+    const { authorizedFetch, rsUseQuery } = useAuthorizedFetch<RSDelivery>();
+    const memoizedDataFetch = useCallback(
+        () =>
+            authorizedFetch(getReportDetails, {
+                segments: {
+                    id: id,
+                },
+            }),
+        [authorizedFetch, id]
+    );
+    return rsUseQuery(
+        // sets key with orgAndService so multiple queries can be cached when viewing multiple detail pages
+        // during use
+        [getReportDetails.queryKey, id],
+        memoizedDataFetch,
+        { enabled: !!id }
```

Comment:
is the !! to convert a value to a boolean? can we be more verbose, and explicitly declare what we want?